readers/postgres/messages.go

```diff
@@ -4,16 +4,23 @@
 package postgres
 
 import (
+	"encoding/json"
 	"fmt"
 
 	"github.com/jmoiron/sqlx" // required for DB access
 	"github.com/mainflux/mainflux/pkg/errors"
+	jsont "github.com/mainflux/mainflux/pkg/transformers/json"
 	"github.com/mainflux/mainflux/pkg/transformers/senml"
 	"github.com/mainflux/mainflux/readers"
 )
 
 const errInvalid = "invalid_text_representation"
 
+const (
+	format   = "format"
+	defTable = "messages"
```

Comment:
Same here, maybe small comment for a code reader.

Reply:
Although here is really looks like you are using `def` for `default`, and maybe comment is not necessary.